Content.Server/Medical/DefibrillatorSystem.cs

```diff
@@ -112,6 +114,14 @@ public bool CanZap(EntityUid uid, EntityUid target, EntityUid? user = null, Defi
         if (!component.CanDefibCrit && _mobState.IsCritical(target, mobState))
             return false;
 
+        if (HasComp<CMDefibrillatorBlockedComponent>(target))
+        {
+            // var text = _entityManager.GetComponent<CMDefibrillatorBlockedComponent>(target).Popup;
+            if (user != null)
+                _popup.PopupEntity(_entityManager.GetComponent<CMDefibrillatorBlockedComponent>(target).Popup, uid, user.Value);
+            return false;
+        }
```

Comment:
This can be changed to a TryComp so that you don't need to get the component twice.
```suggestion
        if (TryComp(target, out CMDefibrillatorBlockedComponent? block))
        {
            if (user != null)
                _popup.PopupEntity(block.Popup, uid, user.Value);
            return false;
        }
```